alkira/resource_alkira_checkpoint_helper.go

```diff
@@ -0,0 +1,194 @@
+package alkira
+
+import (
+	"errors"
+	"log"
+	"strconv"
+
+	"github.com/alkiranet/alkira-client-go/alkira"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type CheckpointGetSegById func(string) (alkira.Segment, error)
```

Comment:
good idea actually. If you want to do it this way, this could be global... We always get segment_id the same way across all.. putting this into global helper. Then you don't need to do this...